connectors/aws/aws-s3/src/main/java/io/camunda/connector/aws/s3/model/request/UploadS3Action.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.aws.s3.model.request;
+
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import io.camunda.connector.generator.java.annotation.TemplateSubType;
+import io.camunda.document.Document;
+import jakarta.validation.constraints.NotBlank;
+
+@TemplateSubType(id = "uploadObject", label = "Upload document")
+public record UploadS3Action(
```

Comment:
We now have 3 names 😄 
id: `uploadObject`, 
label: `Upload document`
record: `UploadS3Action`

Should we go with a single one? Either `PutObject` or `UploadObject`?